README.md

```diff
@@ -72,10 +72,10 @@ include `@glint/environment-ember-template-imports`!)
 
 ## Compatibility
 
-* Ember.js v3.27 or above
-* Ember CLI v3.27 or above
+* Ember.js v4.12 or above
```

Comment:
```suggestion
* Ember.js v3.27 or above
```
we still support and test against 3.27

see here: https://github.com/ember-cli/ember-template-imports/blob/master/config/ember-try.js

Reply:
Thank you, now I understand where else to look in such cases